code/game/objects/structures/stool_bed_chair_nest/chairs.dm

```diff
@@ -274,4 +274,13 @@
 /obj/structure/bed/chair/custom/bar_special
 	name = "bar chair"
 	desc = "Modern design and soft pad. Served up with the drink and great company."
-	icon_state = "bar_chair"
\ No newline at end of file
+	icon_state = "bar_chair"
+
+//onestar
+/obj/structure/bed/chair/custom/onestar
+	name = "onestar chair"
+	desc = "Plain-looking chair. Made by OneStar."
```

Comment:
The description is too plain, even for a plain chair. Make it sound like actual flavor text, like other furniture objects for example.

Reply:
Updated